client/src/app/site/agenda/models/view-item.ts

```diff
@@ -46,6 +46,13 @@ export class ViewItem extends BaseViewModelWithContentObject<Item, BaseViewModel
         return this.item.level;
     }
 
+    /**
+     * @returns An optional subtitle showing for agenda.
+     */
+    public getSubtitle(): string | null {
```

Comment:
Please just do a stub here: ` public getSubtitle: () => string | null;`.
 Add this to the item repo:
```
protected createViewModelWithTitles(model: Item, initialLoading: boolean): ViewItem {
        const viewModel = super.createViewModelWithTitles(model, initialLoading);
        viewModel.getSubtitle = () => this.getSubtitle(viewModel);
        return viewModel;
    }
```